utils/testserver/index.js

```diff
@@ -46,6 +46,7 @@ class TestServer {
     const server = new TestServer(dirPath, port, {
       key: fs.readFileSync(path.join(__dirname, 'key.pem')),
       cert: fs.readFileSync(path.join(__dirname, 'cert.pem')),
+      secureProtocol: 'TLSv1_2_method',
```

Comment:
whats this?

Reply:
Node 12 uses TLS 1.3 by default; we have a few tests that check for the TLS version used. 

This forces TLS to always be 1.2